data/templates/feed-item.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "$root$$url$",
+    "url": "$root$$url$",
+    "content_html": "$description$",
```

Comment:
JSON feed offers two fields for the body: `content` and `content_html`. I think it's important to document that the default template shipped with Hakyll uses `content_html`.

Reply:
Where should that be documented? Is the haddock of `renderJson` a good place for it?

Reply:
Apologies for such a delayed response! Yeah, `renderJson` is the best place for it. To save us both from another review round, I'll update it myself -- I'll have to rebase this PR anyway to apply CI fixes that I'm currently pushing.

Reply:
Done: 1456f3eda398c9f824c7e569bf7f3f03b4697c1a